lib/eventsource.js

```diff
@@ -31,6 +33,8 @@ function hasBom (buf) {
  **/
 function EventSource (url, eventSourceInitDict) {
   var readyState = EventSource.CONNECTING
+  var config = eventSourceInitDict || {}
```

Comment:
I did this to simplify a large number of `eventSourceInitDict && eventSourceInitDict.someProperty` expressions in the rest of this block.